src/consensus/aft/raft.h

```diff
@@ -334,6 +334,11 @@ namespace aft
       return configurations.back().nodes;
     }
 
+    uint32_t node_count() const
+    {
+      return nodes.size();
```

Comment:
We have to be careful with this, because this is the union of all currently active nodes, it is not necessarily equal to the number of nodes in the current configuration. If for example we're reconfiguring from a 4 node network to another 4 node network, this could be 7 (or any value between 5 and 8).